src/atoms/TagLink/tests/TagLink.test.js

```diff
@@ -5,28 +5,26 @@ import TagLink from '../TagLink';
 // automatically unmount and cleanup DOM after the test is finished.
 afterEach(cleanup);
 
-describe('Taglink tests', () => {
-  test('TagLink props value matches as expected', () => {
-    // Arranged
-    const tag = {
-      href: '/the/url/path',
-      tagName: 'Live from Here'
-    };
-    // Act
-    const { getByText } = render(
-      <TagLink tagName={tag.tagName} href={tag.href} />
-    );
-    const TagLinkNode = getByText('Live from Here');
-    // Assert
-    expect(TagLinkNode.getAttribute('href')).toBe('/the/url/path');
-    expect(TagLinkNode.innerHTML).toBe('Live from Here');
-  });
-  test('Throws an error when a isRequired value is missing', () => {
-    expect(() => {
-      render(<TagLink />);
-    }).toThrow();
-  });
-  test("We'll have to overhaul TagLink later anyway", () => {
-    expect(true).toBe(true);
-  });
+test('TagLink props value matches as expected', () => {
+  const tag = {
+    href: '/the/url/path',
+    tagName: 'Live from Here'
+  };
+
+  const { getByText } = render(
+    <TagLink tagName={tag.tagName} href={tag.href} />
+  );
+  const node = getByText('Live from Here');
+
+  expect(node).toBeDefined();
+  expect(node.getAttribute('href')).toBe('/the/url/path');
+  expect(node.innerHTML).toBe('Live from Here');
+});
+test('Throws an error when a isRequired value is missing', () => {
+  expect(() => {
+    render(<TagLink />);
+  }).toThrow();
+});
+test("We'll have to overhaul TagLink later anyway", () => {
```

Comment:
this test should probably be deleted. It's kind of there as a note, but wasn't followed up on, and is not really testing anything.